src/Files.App/UserControls/StatusCenter.xaml

```diff
@@ -740,6 +69,420 @@
 				</Style>
 			</ListView.ItemContainerStyle>
 
+			<ListView.ItemTemplate>
+				<DataTemplate x:DataType="statuscenter:StatusCenterItem">
+					<UserControl>
+						<Grid
+							x:Name="ContentRoot"
+							Padding="8,8,8,8"
+							HorizontalAlignment="Stretch"
+							AutomationProperties.Name="{x:Bind Header, Mode=OneWay}"
+							Background="{StaticResource CardBackgroundFillColorDefaultBrush}"
+							BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}"
+							BorderThickness="1"
+							CornerRadius="{StaticResource OverlayCornerRadius}">
+							<Grid.RowDefinitions>
+								<RowDefinition Height="Auto" />
+								<RowDefinition Height="Auto" />
+							</Grid.RowDefinitions>
+
+							<Grid.Transitions>
+								<TransitionCollection>
+									<RepositionThemeTransition />
+								</TransitionCollection>
+							</Grid.Transitions>
+
+							<!--  Header  -->
+							<Grid x:Name="StatusCenterItemHeaderPanel" Grid.Row="0">
+								<Grid.ColumnDefinitions>
+									<ColumnDefinition Width="Auto" />
+									<ColumnDefinition Width="*" />
+									<ColumnDefinition Width="Auto" />
+								</Grid.ColumnDefinitions>
+
+								<!--  Header Icon  -->
+								<Grid Grid.Column="0">
+
+									<!--  Circle Icon 32x32  -->
+									<TextBlock
```

Comment:
What's this used for?

Reply:
Literally circle icon. This circle is made with an icon not an Ellipse.

![image](https://github.com/files-community/Files/assets/62196528/9e1ba505-d04a-4cbb-bb39-c7ac7766a25b)

Reply:
Why don't you use border?

Reply:
Is the circle icon fully opaque?  Some of the Themed Icons have opacity per layer.

Reply:
Right I will use a Border for it.